docs/authentication/saml/account-linking.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking during SAML SSO.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both SAML SSO and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an SAML provider returns an email address that matches an existing account, assuming a single owner for each email address.
+
+## How it works
+
+When a user attempts to sign in or sign up, Clerk first checks the email address provided. Clerk will attempt to link the SAML account with any existing Clerk account that shares the same email address.
+
+In the following sections, we'll look at the different scenarios that can occur during this process and explain how Clerk handles each one.
+
+<Callout type="info">
+    Email addresses from SAML providers are considered verified by default!
```

Comment:
```suggestion
    Email addresses from SAML providers are considered verified by default.
```

Lovely note ❤️ Let's avoid exclamation points though (I know they exist elsewhere in the docs, but I'd like to get rid of them over time)